### PR TITLE
Fixes on api updating database on create.

### PR DESCRIPTION
**Pull request by AldrenKim**
Fixed models for both plant and animals.
Added a trailing '/' for both api urls.

### CODE REVIEW

File: client/src/pages/CreateRecord/View/index.tsx [hunk 1]
```diff
@@ -44,34 +46,42 @@ export default function View() {
...
 
   const onFinish = async (values: any) => {
-    const fetchedData = await addAnimal(axios, values);
+    let fetchedData;
+    if (typeOfItem === 'Animals') {
+      fetchedData = await addAnimal(axios, values);
+    } else {
+      fetchedData = await addPlant(axios, values);
+    }
     console.log(fetchedData);
```
Comment (e6654321) [inline client/src/pages/CreateRecord/View/index.tsx:55]: add message prompt for success then navigate back to table

File: client/src/pages/CreateRecord/View/index.tsx [hunk 2]
```diff
@@ -44,34 +46,42 @@ export default function View() {
...
   };
 
   const onReset = () => {
     form.resetFields();
   };
 
-  const onFinishFailed = (values: any) => {
-    console.log('Click failed', values);
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed', errorInfo);
```
Comment (e6654321) [inline client/src/pages/CreateRecord/View/index.tsx:63]: use message prompt for error

**Review by e6654321 — CHANGES_REQUESTED** [review]

File: client/src/pages/CreateRecord/View/index.tsx (hunk 1 above)
Comment (AldrenKim) [reply client/src/pages/CreateRecord/View/index.tsx:55]: Noted.

Comment (AldrenKim) [reply client/src/pages/CreateRecord/View/index.tsx]: Resolved with commit 109dc50

File: client/src/pages/CreateRecord/View/index.tsx (hunk 2 above)
Comment (AldrenKim) [reply client/src/pages/CreateRecord/View/index.tsx]: Resolved with e1c485a

**Review by e6654321 — APPROVED** [review]
Follow new structure

**Pull request merged**